app/assets/javascripts/snowcrash/pages/nodes/new_form.js

```diff
@@ -0,0 +1,38 @@
+(function() {
+  var copyOver = function($to, from, $typeTo, $typeFrom) {
+    if ($to.val() === '' && from !== '') {
+      $to.val(from.split('\n')[0] + '\n');
```

Comment:
Minor suggestion. If we add `trim()` here:
```suggestion
      $to.val(from.trim().split('\n')[0] + '\n');
```

We can catch the first element in a text area that looks like this:
```


10.0.0.1
```
which would replace the text input with `10.0.0.1`